kefir.js.flow

```diff
@@ -46,6 +46,8 @@ declare class Observable<+V,+E=*> {
   offLog(name?: string): this;
   spy(name?: string): this;
   offSpy(name?: string): this;
+  setName(name?: string): this;
+  setName(Observable<V, E>, name?: string): this;
```

Comment:
I think this should be `setName(Observable<any, any>, name: string): this` i.e., say an `Observable` of `number` can be created from an `Observable` of another type, not necessarily `number`. And  `name` also not optional.